src/components/CustomerQuota/ItemsSelection/AddonsItems.tsx

```diff
@@ -1,13 +1,72 @@
-import React, { FunctionComponent } from "react";
-import { ActivityIndicator } from "react-native";
+import React, { FunctionComponent, useContext, useEffect } from "react";
+import { ActivityIndicator, View } from "react-native";
 import { color } from "../../../common/styles";
+import { AlertModalContext } from "../../../context/alert";
+import { AuthContext } from "../../../context/auth";
+import { CampaignConfigContext } from "../../../context/campaignConfig";
+import { usePastTransaction } from "../../../hooks/usePastTransaction/usePastTransaction";
+import { useTranslate } from "../../../hooks/useTranslate/useTranslate";
+import { TransactionsGroup } from "../TransactionsGroup";
+import {
+  groupTransactionsByCategory,
+  sortTransactionsByCategory,
+} from "../utils";
 
-export const AddonsItems: FunctionComponent = () => {
-  return (
+const BIG_NUMBER = 99999;
+
+export const AddonsItems: FunctionComponent<{
+  ids: string[];
+  isShowAddons: boolean;
+}> = ({ ids, isShowAddons }) => {
+  const { sessionToken, endpoint } = useContext(AuthContext);
+  const { policies: allProducts } = useContext(CampaignConfigContext);
+  const { pastTransactionsResult, loading, error } = usePastTransaction(
```

Comment:
Can we do the following to avoid the need for line 29?
```suggestion
  const { pastTransactionsResult: sortedTransactions, loading, error } = usePastTransaction(
```